build-script.py

```diff
@@ -376,6 +397,35 @@ def build(self, product_name):
         check_call(command, env=env, verbose=self.verbose)
 
 
+# -----------------------------------------------------------------------------
+# Running
+
+
+def run_code(runner, arguments, toolchain, build_dir, multiroot_data_file, release, verbose):
```

Comment:
I think we can just inline this to `run_code_generation` because `run_code_generation` is currently very simple and this is also pretty straight-forward straight-line code.

Reply:
So we should remove `run_code ` and move all the logic into `run_code_generation`?

Reply:
That’s what I thought